build.gradle

```diff
@@ -71,3 +74,64 @@ dependencies {
         //implementation 'org.springframework:spring-web:5.0.2.RELEASE'
     }
 }
+
+def classFilesArray = []
+def execFilesArray = []
+
+task copyExecFilesAndJavaClassFiles {
+    subprojects.forEach { subproject -> 
+        def execFile = new File("${subproject.buildDir}/jacoco/jacoco.exec")
+        if (execFile.exists()) {
+            execFilesArray.push(execFile)
+            fileTree("${subproject.buildDir}/classes").matching {
+                exclude '**/test/*'
+                exclude '**/module-info.class'
+                include '**/*.class'
+            }.each { file -> classFilesArray.push(file) }
+        }
+    }
+}
+
+task copyBallerinaClassFiles {
+    subprojects.forEach { subproject -> 
+    def ballerinaSourceDirectory = new File("${subproject.buildDir}/ballerina-src")
+        if (ballerinaSourceDirectory.exists() && subproject.projectDir.getParentFile().getName() == "langlib") {
+            fileTree("${subproject.buildDir}/ballerina-src/target").include("**/*.zip").forEach { zip ->
+                def jarFiles = zipTree(zip).matching { include '**/*.jar' } 
+                jarFiles.forEach { jar ->
+                    zipTree(jar).matching {
+                        exclude '**/tests/*'
+                        exclude '**/$_init.class'
+                        exclude '**/$value$Caller.class'
+                        exclude '**/$value$Detail.class'
+                        exclude '**/$value$DetailType.class'
+                        exclude '**/$value$EmptyIterator.class'
+                        exclude '**/$value$$anonType$_6.class'
+                        exclude '**/$value$$anonType$_*.class'
+                        exclude '**/$value$_Frame.class'
+                        include '**/*.class'
+                    }
+                    .each { file -> classFilesArray.push(file) }
```

Comment:
Shall we take this to the previous line?

Reply:
I changed. Commit is Id is 6d9d660229af842f3af0e207eb792fbe96b416e1